server/polar/customer/repository.py

```diff
@@ -101,6 +101,18 @@ async def get_by_stripe_customer_id_and_organization(
         )
         return await self.get_one_or_none(statement)
 
+    async def stream_by_organization(
+        self,
+        auth_subject: AuthSubject[User | Organization],
+        organization_id: Sequence[UUID],
```

Comment:
Actually 🤓 I saw mypy complaining because `organization_id` can actually be `None` from the request. So the solution is to handle that here as well:

```py
organization_id: Sequence[UUID] | None,
) -> AsyncGenerator[Customer]:
    # ...
    if organization_id is not None:
        statement = statement.where(Customer.organization_id.in_(organization_id))
``